mp/src/game/client/momentum/ui/leaderboards/ClientTimesDisplay.cpp

```diff
@@ -176,7 +181,7 @@ void CClientTimesDisplay::Close()
     SetMouseInputEnabled(false);
     SetKeyBoardInputEnabled(false);
 
-    g_pClientMode->GetViewportAnimationController()->StartAnimationSequence(GetParent(), "LeaderboardsBgFocusLost");
+    g_pClientMode->GetViewportAnimationController()->StartAnimationSequence(this, "LeaderboardsBgFocusLost");
```

Comment:
Not sure on this change, shouldn't this be `StartAnimationSequenceForPanel` ?

Reply:
Hmm, I tried `GetAnimationController()->StartAnimationSequenceForPanel` but that failed spectacularly by extending the panel to the full screen size and then throwing a SIGFPE. 
Is the current way wrong because it's not a Viewport Panel anymore?

Reply:
Huh, okay. I thought the animation was doing something in this class (CClientTimesDisplay) but it looks like this would be correct. 👍

Reply:
I guess I can still use `GetAnimationController()->StartAnimationSequence(...)` to skip that indirection though

Reply:
So long as the animation still works with this change, it's okay. I was just curious if the animation broke at all when doing this.